### ISSUE TITLE
Tapping on a multi-line text box in a WKWebView makes the focus jump out of the webview and into Flutter

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#65027]
Unfortunately, there's no straight-forward repro for this. You can:

- Turn on voice over on iOS.
- Install an app that contains a Flutter scaffold + appbar with action buttons. The body of the scaffold should be webview_flutter plugin.
- Via webview_flutter plugin, open up a webpage which contains a multi-line text edit box.
- Focus on the text box via swipes.
- Double tap to activate the text box. You will notice the keyboard will show and the focus will jump to one of the action buttons in the appbar.

Expected behavior is that focus stays on the text box.

**Comment by chunhtai** [comment]
cc @blasten

**Comment by Hixie** [comment]
This is P2 because it will be a launch blocker for customer: money.

**Comment by chinmaygarde** [comment]
@cyanglaz: You seem to have a ton of high priority issues on your plate. Can you update your assignments to indicate the items you are actively working on? @zanderso will also reach out to help with redistribute the workload.

**Comment by cyanglaz** [comment]
@chinmaygarde This issue is on my queue right after the flaky test crash issue. The flaky test is taking up all of my time these days. I will un-assign myself for now until I can actively work on this.

**Comment by chinmaygarde** [comment]
cc @cyanglaz: Will you be able to get to this after your work on the image picker crash?

**Comment by cyanglaz** [comment]
@chinmaygarde Yes this is the next one in my queue.

**Comment by chinmaygarde** [comment]
Reducing priority per guidance in the linked internal bug.

**Comment by chinmaygarde** [comment]
@mehmetf: I don't see any relevant updates to the internal bug that would necessitate a reevaluation of priorities. Can you add more context please?

**Comment by mehmetf** [comment]
p0, p1, p2's are synced periodically with buganizer being the source of truth. Since this was lowered to P4, we should have lowered the buganizer bug to p3 (or below).

I will change the tool to provide more context on such updates.

**Comment by chinmaygarde** [comment]
> we should have lowered the buganizer bug to p3

Can the bot do this for us? This seems like we can automate easily. In fact, it is already being done from Buganizer to GitHub. Just go the other way around. Whichever is updated later takes precedence. I'd rather keep the issue triages simple and limited to one tracker.

**Comment by mehmetf** [comment]
I prefer having a single source of truth for bugs that are filed in Google. That's buganizer. We have seen cases in the past where github issue priority was updated incorrectly (that's one of the reasons we have created the sync bot).

To help this problem, when the bot updates priority, it will also post a comment explaining why the priority was bumped (with a bug link). Additional synchronization can be done if so desired.
